### ISSUE TITLE
Apply proper roles to status messages

### ISSUE THREAD

**Issue by adriannelee** [cds-snc/notification-planning#567]
Issue group: Banners
Visibility: 2
Evaluation: Does not support
Success Criteria: 4.1.3: Status Messages
Needs design: 

### Description
Success messages should have the [status](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role) role. 

![](https://dl.airtable.com/.attachments/b2791363f577009ac5ff6130d0d3be3d/439409c8/Capturedecranle2022-03-15a14_01_17.png)

**Comment by amazingphilippe** [comment]
Fixed by https://github.com/cds-snc/notification-admin/pull/1253
